### PR TITLE
RSP-1914 language translations

### PR DESCRIPTION
**Pull request by mgoldson**
Adds new language translations for users of the site:

- New language links in the footer
- New languages are added to whitelist
- New languages added to Cypress e2e translations test

[dev01_translations_screenshots.zip](https://github.com/dvsa/rsp-public-portal/files/5475120/dev01_translations_screenshots.zip)

### CODE REVIEW

File: src/server/app.js [hunk 1]
```diff
@@ -98,7 +98,31 @@ export default async () => {
...
+      'ee',
+      'gr',
+      'hu',
+      'it',
+      'lv',
+      'lt',
+      'pt',
+      'ro',
+      'ru',
+      'rs',
```
Comment (grjacob) [inline src/server/app.js:121]: Is this meant to be Serbian? I think it should be 'sr' if so.

File: src/server/i18n/bg.json [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    "cookie_preferences": "Cookie preferences",
+    "cookie_details": "Cookie details"
+  },
+  "cookie_banner": {
+    "title": "Tell us whether you accept cookies",
+    "message": "We use <a href='/cookie-details'>cookies to collect information</a> about how you use the Roadside Payments service. We use this information to make the website work as well as possible and improve government services.",
+    "button_accept": "Accept all cookies",
+    "button_preferences": "Set cookie preferences"
+  },
+  "languages": {
```
Comment (grjacob) [inline src/server/i18n/bg.json:25]: What is this list used for? Should it contain the new languages?

File: src/server/i18n/bg.json [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+    "cookie_preferences_cookie_name_one": "cm-user-preferences",
+    "cookie_preferences_cookie_purpose_one": "Saves your cookie consent preferences",
+    "cookie_preferences_cookie_expires_one": "1 year",
+    "change_preferences_title": "Change your preferences",
+    "change_preferences_text": "You can <a href='/cookie-preferences'>change which cookies you’re happy for us to use.</a>",
+    "table_header_name": "Name",
+    "table_header_purpose": "Purpose",
+    "table_header_expires": "Expires"
+  }
+}
```
Comment (grjacob) [inline src/server/i18n/bg.json:241]: No new line at end of file

Comment (grjacob) [reply src/server/i18n/bg.json:241]: Please can you check the other files for this too

File: cypress/integration/translations.feature [hunk 4]
```diff
@@ -12,11 +12,28 @@ Feature: Language Translations
...
+    | Payment code | Estonian         | ee            |
+    | Payment code | Greek            | gr            |
+    | Payment code | Hungarian        | hu            |
+    | Payment code | Italian          | it            |
+    | Payment code | Latvian          | lv            |
+    | Payment code | Lithuanian       | lt            |
+    | Payment code | Portugeuse       | pt            |
+    | Payment code | Romanian         | ro            |
+    | Payment code | Russian          | ru            |
+    | Payment code | Serbian-Cyrillic | rs            |
```
Comment (grjacob) [inline cypress/integration/translations.feature:33]: Think this should be 'sr'.

File: src/server/i18n/bg.json (hunk 2 above)
Comment (mgoldson) [reply src/server/i18n/bg.json:25]: It should do but this is only appears to be used on the start page which is not used. Also, we don't have the translations for each new language in the language of each new language anyway (if that makes sense)

File: src/server/app.js (hunk 1 above)
Comment (mgoldson) [reply src/server/app.js:121]: The 2 character languages codes are based on the list here: https://www.iban.com/country-codes. SR would be used for Suriname

File: src/server/i18n/bg.json (hunk 3 above)
Comment (mgoldson) [reply src/server/i18n/bg.json:241]: Sure, will do.

File: cypress/integration/translations.feature [hunk 5]
```diff
@@ -12,11 +12,28 @@ Feature: Language Translations
...
-    | Payment code | Polish   | pl            |
-    | Payment code | Welsh    | cy            |
+    | page name    | language         | language code |
+    | Payment code | French           | fr            |
+    | Payment code | German           | de            |
+    | Payment code | Polish           | pl            |
+    | Payment code | Welsh            | cy            |
+    | Payment code | Bulgarian        | bg            |
+    | Payment code | Croatian         | hr            |
+    | Payment code | Czech            | cz            |
```
Comment (grjacob) [inline cypress/integration/translations.feature:22]: Should be 'cs' https://en.wikipedia.org/wiki/Czech_language

File: cypress/integration/translations.feature [hunk 6]
```diff
@@ -12,11 +12,28 @@ Feature: Language Translations
...
+    | Payment code | Dutch            | nl            |
+    | Payment code | Estonian         | ee            |
```
Comment (grjacob) [inline cypress/integration/translations.feature:24]: Should be 'et' https://en.wikipedia.org/wiki/Estonian_language

File: cypress/integration/translations.feature [hunk 7]
```diff
@@ -12,11 +12,28 @@ Feature: Language Translations
...
+    | Payment code | Greek            | gr            |
```
Comment (grjacob) [inline cypress/integration/translations.feature:25]: Weirdly, Greek is 'el'! https://en.wikipedia.org/wiki/Greek_language

**Review by grjacob — APPROVED** [review]

**Review by ammarhaiderbjss — APPROVED** [review]
Looks good

**Pull request merged**